### DISCUSSION TITLE
setting date in tinacms and hugo

### DISCUSSION THREAD

**Discussion by levanthanhlong** [tinacms/tinacms#4767] [Help]
hello! The date of the article saved to the markdown file in hugo is 1 day later than the date selected in tinaCMS. How can i fix it?

**Comment by JackDevAU** [comment]
Hey @levanthanhlong 

This is due to the date component not using utc time and instead using local time. Have a look at what Tina.io is doing to fix this issue https://github.com/tinacms/tina.io/pull/2147. Hope that helps
